tekton/ci/custom-tasks/pr-commenter/pkg/reconciler/reconciler.go

```diff
@@ -163,6 +163,16 @@ func createEntry(report *ReportInfo) string {
 }
 
 func (c *Reconciler) reportComment(ctx context.Context, report *ReportInfo, logger *zap.SugaredLogger) error {
+	// First, check if the PR is still open. If it isn't, don't comment.
+	pr, _, err := c.SCMClient.PullRequests.Find(ctx, report.Repo, report.PRNumber)
```

Comment:
Hopefully in future we can work with GitHub proxy to avoid too many outgoing API request